### ISSUE TITLE
One person, with multiple IDs

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3409]
**One person, with multiple IDs**

Recorded by `Bjørn Petter Løfall`.
Recorded By ID `https://www.wikidata.org/wiki/Q21519416 | https://orcid.org/0000-0001-9645-3394`

Both the Wikidata and the Orcid is `Bjørn Petter Løfall`

What is the recommended usage of this field? To provide all known IDs per person or to provide ONE ID per recorder?

https://www.gbif.org/occurrence/1702184576

-----

Github user: @MortenHofft
User: [See in registry](https://www.gbif.org/api/feedback/user/e2af1f2a0975e0b017c8ec98e54e1f63:3a8d433c76c473689ceabe4b4159922cca0171dd4bea0b77b5b7b0bd88bbbd560e5aeae87576f9627b0fec936f13d1a0ccf0aa2854a718d966c64e5bb3ecd89a)
System: Chrome 90.0.4430 / Mac OS X 10.15.7
Referer: https://www.gbif.org/occurrence/1702184576
Window size: width 1440 - height 766
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-05-04T12:29:18.829Z',to:'2021-05-04T12:35:18.829Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2021-05-04T12:29:18.829Z',to:'2021-05-04T12:35:18.829Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL
datasetKey: e45c7d91-81c6-4455-86e3-2965a5739b1f
publishingOrgKey: f314b0b0-e3dc-11d9-8d81-b8a03c50a862

**Comment by MattBlissett** [comment]
"An unordered list (concatenated and separated) of IDs representing names of people, groups, or organizations responsible for recording the original Occurrence. No semantics should be assumed, including for example an ordering of identifiers to indicate a primary collector or any institutional affiliation."

So it's fine to do this.

**Comment by dshorthouse** [comment]
Technically ok, but not very helpful downstream in practice. I'd not recommend that the same person be referred to with multiple identifiers but that one be used given there might be a choice from the provider. The reason is somewhat implicit in "no semantics should be assumed", which would include in this instance, "sameAs".

The proposed [`recordedByID`](https://github.com/tdwg/dwc/issues/102) term is defined as w/ comment:

**Definition:** A list (concatenated and separated) of the globally unique identifier for the person, people, groups, or organizations responsible for recording the original Occurrence.
**Comment:** The order of the identifiers on any list for this term can not be assumed to convey any semantics. Recommended best practice is to provide a single identifier that disambiguates the details of the identifying agent and to separate the values in a list with space vertical bar space ( | ).

**Comment by rukayaj** [comment]
I just found this after posting https://github.com/gbif/portal-feedback/issues/5055 - if there's not supposed to be any meaning in the order of recordedByID and it doesn't need to relate to the recordedBy order then my issue can get closed. It's a bit confusing for people though.

**Comment by dshorthouse** [comment]
> I just found this after posting #5055 - if there's not supposed to be any meaning in the order of recordedByID and it doesn't need to relate to the recordedBy order then my issue can get closed. It's a bit confusing for people though.

Agreed and the qualification certainly was not without debate. The trouble with a pipe-separated list is that it also cannot express completeness such as when the primary collector has no known identifier whereas some of the team members in a collecting event do. You could make an attempt at solving this as a data publisher with positional pipes w/o intervening values but that's terribly messy. You'd be overloading the term, trying to communicate roles to a consumer that could be common in some circles, less so in others but then the presentation gives the appearance of a technical glitch.
